src/app/themes.tsx

```diff
@@ -152,7 +152,9 @@ interface Props {
 
 export const ThemeUpdateContext = createContext((dark: boolean): void => {
     throw new Error(
-        `attempted to set theme outside of a ThemeUpdateContext.Provider: Dark: ${dark}`
+        `Attempted to set theme to ${
```

Comment:
is this something shown to users? (what do you think of having a message on error pages for users only (user friendly) and maybe a troubleshooting icon that opens up to dev-friendly error messages?)

Reply:
It is, but also should never really happen? Tbh I'm not sure when it would ever come up, but also it was there initially so I didn't want to remove it!

Reply:
@SamYap-Softwire we generally don't want this, usually in production something like this would use a library like Sentry (https://docs.sentry.io/platforms/javascript/) so that we know when and where exceptions happen. (This error specifically should never be shown to users, we have a context provider at the top of the tree)